.env.example

```diff
@@ -1,14 +1,22 @@
-## get from AWS
+# get from AWS
 AWS_ACCESS_KEY_ID=AKIA...
 AWS_SECRET_ACCESS_KEY=...
-## need to be created
+# need to be created
 ROLE_ARN=
 
-## get from Seller Central / Developer Central account
+# get from Seller Central / Developer Central account
 CLIENT_ID=
 CLIENT_SECRET=
-## self authorize
+# self authorize
 REFRESH_TOKEN=
 
-## set to your preference
+# set to your preference
 API_REGION=eu-west-1
+
+# authorize github api when check api model commits
+GITHUB_TOKEN=
+
+# api model's generation frequency.
+# Format: {hours}, {minutes}, {seconds}, {milliseconds}
+# Default: -24, 0, 0, 0
```

Comment:
I don't think we need this much granularity. Just hours is enough.

Should be a positive hours number. Even the name says so: `HOUR_FREQUENCY` ;)

Although should probably be `UPDATE_FREQUENCY_HOURS` to be more clear.

Reply:
@moltar : 
> Should be a positive hours number
Why should it be a positive hours number?
Because this use to show updated after the given time. 🤔 
I think we are expected values are 0-23. Or a negative hours number to show updated from previous days

Reply:
The setting is for humans, which says "frequency", frequency can never be negative.

If this is not frequency, then let's rename the variables. If this is the time we are looking back, then maybe call it `LOOKBACK_HOURS`, in which case it still should be positive, as it is for humans.

But in the use case, you can multiple by `-1`, if you need to.

These are just different interfaces. Human interfaces are human friendly, and machine interfaces are for machines and they can adapt to humans, not the other way around :)